package.json

```
@@ -10,7 +10,13 @@
     "prepare": "husky"
   },
   "dependencies": {
+    "@radix-ui/react-progress": "^1.1.7",
```

Biome (2.1.2)

[error] The key @radix-ui/react-progress was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)